gui-apps/foot-terminfo/foot-terminfo-1.10.3.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz  -> foot-${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/foot"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/foot.git"
+fi
+
+DESCRIPTION="Terminfo for foot, a fast, lightweight and minimal Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+LICENSE="MIT"
+SLOT="0"
+
+DEPEND=""
+RDEPEND="<sys-libs/ncurses-6.3"
+BDEPEND="sys-libs/ncurses"
+
+src_prepare() {
+	default
+	sed -i 's/@default_terminfo@/foot/g' "${S}/foot.info" || die "Failed to pre-process terminfo"
```

Comment:
`${S}` is the current directory, so there's no need to specify it, same for other cases below.

Also I personally usually skip die message for a mostly self-explanatory line (given portage displays the line it died on).